openy.profile

```diff
@@ -269,6 +269,30 @@ function openy_discover_broken_paragraphs(array &$install_state) {
       $query->execute();
     }
   }
+
+  $schedule_paragraph_tables = ['paragraph__field_prgf_schedules_ref', 'paragraph_revision__field_prgf_schedules_ref'];
```

Comment:
The following code should handle the broken paragraphs, though this is not new to this PR. I was not able to replicate that issue today.